src/components/Feedback.tsx

```diff
@@ -106,10 +131,10 @@ export const Feedback: React.FC<Props> = ({
                         {...formik.getFieldProps('whatWentWrong')}
                       />
                     </div>
-                    {/* <CheckboxSmall
+                    <CheckboxSmall
                       name="agree"
-                      label="Suhlasím s odoslaním dát ktoré som vyplnil (zatiaľ nie anonymne)"
-                    /> */}
+                      label="Suhlasím s odoslaním anonymných dát ktoré som vyplnil/a"
```

Comment:
Mame k tomuto copy? Ja ako user by som rad vedel co konkretne sa posiela. Prip vysvetlit na ake ucely sa to posiela?

```
Spolu s formularom sa odoslu vsetky udaje ktore ste vyplnili v aplikacii okrem osobnych udajov (meno, priezvisko, atd, doplnit). Tieto informacie nam pomozu lepsie pochopit chybu na ktoru ste narazili a opravit ju.
```

Reply:
Nemame oficialne copy, mozem spravit trello task. Zatial to beriem ze je to primarne pre nase interne testovanie.